### PR TITLE
add call of after handler for cancelled requests

### PR DESCRIPTION
**Pull request by malaupa**
* complete cancel subject for finished requests

### CODE REVIEW

File: src/mustache/api.service.mustache [hunk 1]
```diff
@@ -339,8 +341,6 @@ export class {{classname}} {
         if (cancelPreviousRequest) {
             if (this.cancelMap['{{operationIdOriginal}}']) {
                 this.cancelMap['{{operationIdOriginal}}'].next();
-                this.cancelMap['{{operationIdOriginal}}'].complete();
```
Comment (amjmhs) [inline src/mustache/api.service.mustache]: Why is there a different handling in code for http-client and not-http-client? (comparing line 410-413)

File: src/mustache/api.service.mustache [hunk 2]
```diff
@@ -351,12 +351,30 @@ export class {{classname}} {
...
+          }
+          if (typeof this.configuration.afterHandler === 'function') {
+            this.configuration.afterHandler(
+              '{{operationIdOriginal}}',
+              '{{httpMethod}}'.toUpperCase(),
+              result ? result : new Error('CANCELLED')
+            );
+          }
+        };
+        handle = handle.pipe(share());
```
Comment (amjmhs) [inline src/mustache/api.service.mustache]: should be the line 352 after this one ?!
`handle = handle.pipe(takeUntil(this.cancelMap['{{operationIdOriginal}}']));`

Comment (malaupa) [reply src/mustache/api.service.mustache]: No.

File: src/mustache/api.service.mustache (hunk 1 above)
Comment (malaupa) [reply src/mustache/api.service.mustache]: Done!

**Pull request merged**